ops/ansible/playbooks-ccs/files/test-bluebutton-appserver-public-cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
+MIIFQjCCAyqgAwIBAgIEb52qHjANBgkqhkiG9w0BAQsFADAbMRkwFwYDVQQDExB0
+ZXN0LmJmZC5jbXMuZ292MB4XDTE5MDkxODAyMDI0MFoXDTIxMDkxNzAyMDI0MFow
+GzEZMBcGA1UEAxMQdGVzdC5iZmQuY21zLmdvdjCCAiIwDQYJKoZIhvcNAQEBBQAD
+ggIPADCCAgoCggIBAJXx5y7OuPH4Jd1nPQonubCDgQJB4gwiMxgDCRd9bYKnYFJN
+aC41t3cR9L5uN5Kgu3FLpWopa/pV2oczYzHSMC7PlCvMHT/sAuNyBoDq+CkqbZdc
+60Y69ta8YT6PRTUYXMz+ypVIC2glmH+3NhLPK1dE/8i2LO3HoY+TGogL+vSVV2dO
+UEOwerEAFIRUJPkqMKbMBHXUA9utxSOzah7mlUNstYSt2kY+CaLFOnYwN/PwQ0/c
+cumLXTyg3cRluv7HQdMSYHpgOdXOdJ6dYrq6fu0Wb3TRqs/xzAlUb42ttwbGdB3p
+D/f8mxmwJl9SccVhpthiidsoY9eBj76q/KihKt1Rg+if9q7g7/TPIiZ9LTw5PNRM
+wG1DFMwtRTVsx0aeH6+pmnP5/1rEBlsKLg2eSN2P1BTRuaVM0rf0zs2ZD+oa2q4l
+EE6vdeBexP+q/A5NrSlP19GfCjOuerSL5aQVjCe9OsMUzDD8R1+OqIP6kG5rhQ7w
+LejndjxUyZ3QG5vx/0ULkDSzKOGjlfVkx3Jb8SvkxtPJqVSi4+CteuMwDmU9yFnv
+yGSbXOtOIwiF/CyagB2LDzFtH0WqUppkuAraTSm8ngFnD6FH0xd37A0FGs4YRExi
+VUqMvncd0uI+C82Jt0VfCvpKGCLBxb2fkF8pbiTHDdAkFPmheD03kARhN8wDAgMB
+AAGjgY0wgYowaQYDVR0RBGIwYIIRdGVzdC5iZmRjbG91ZC5uZXSCOWludGVybmFs
+LXRzYmIxMGxiMDEtNzU4ODU1MjM2LnVzLWVhc3QtMS5lbGIuYW1hem9uYXdzLmNv
+bYIQdGVzdC5iZmQuY21zLmdvdjAdBgNVHQ4EFgQUeYpLunFjiLThTfxYovZJb4U/
++UQwDQYJKoZIhvcNAQELBQADggIBAAgPjJEZ7Attl+E95wZxqvwJiE33KIVwdJhF
+SjMk/nXwT/RAtZzO5cxpsYrV5KrILYSeuB376ASENVFCt3g5swRouxActCL5YP8s
+4VaFYIU964/K6YoOhm8tizFVqiTg8vPLYtv7XEhpw2qUjDtwkA64T273AF/SvJ38
+mBpwonDc1U7dHjwL2oC/hIxWV1l+d2AF70iiUCAtFt3xCNCTSIZCfAv4hPXkr3UZ
+ThTqULrHs+y/F5qrrTPZ3/6Fc/u4dKZ3EtdsivsCsoePzkvKHo6ToNTbw+BRwZnm
+upnj+K0PSV63TtC01BHdJ97LkRODxwjQ22Y5yJm/dI8k9IemSt7vvv99oZhqLPx7
+dITdigCb5D1zGtymx/vgrZSoZ8VXZDWpfXqotTr6CbvnQeWuvG+Qp2buaZZ9qxmR
+MNMELIvyoDtL0ZeGwxpg8WhumUKCwVgtjBNxg4UYjTihACsKFcDmVSIaDswuFKAL
+2tpsnIoZV/YMuE6MKzTG8GgKPWLr95Pe4yHWKtkdUqvnFranHvc6jMc/ExVvPhLX
+YVKfEfwZoncSuj/f3EnqFSwB7Mdb3lyu6b6pMywvtqDuiX9RcxPxdH5V7pAGcKb7
+5yKiuc61sXkFSApvp9Q5c6psFe4PfuiuWe4tr4uDigJkbo8hjpRdjehhmz/Yy2PS
+nZfp/KzL
+-----END CERTIFICATE-----
```

Comment:
Reviewing the cert:

```
$ openssl x509 -text -in ops/ansible/playbooks-ccs/files/test-bluebutton-appserver-public-cert.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 1872603678 (0x6f9daa1e)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=test.bfd.cms.gov
        Validity
            Not Before: Sep 18 02:02:40 2019 GMT
            Not After : Sep 17 02:02:40 2021 GMT
        Subject: CN=test.bfd.cms.gov
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:95:f1:e7:2e:ce:b8:f1:f8:25:dd:67:3d:0a:27:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Alternative Name:
                DNS:test.bfdcloud.net, DNS:internal-tsbb10lb01-758855236.us-east-1.elb.amazonaws.com, DNS:test.bfd.cms.gov
            X509v3 Subject Key Identifier:
                79:8A:4B:BA:71:63:88:B4:E1:4D:FC:58:A2:F6:49:6F:85:3F:F9:44
    Signature Algorithm: sha256WithRSAEncryption
         08:0f:8c:91:19:ec:0b:6d:97:e1:3d:e7:06:71:aa:fc:09:88:
...
```

Looks good to me.

For comparison, here's the old test cert (note: the version we had in Git was wrong; I'm about to fix that in a commit on your branch):

```
$ openssl x509 -text -in ops/ansible/playbooks-healthapt/files/server_data_server_env_test_certificate.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 579344056 (0x228816b8)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=internal-tsbb10lb01-758855236.us-east-1.elb.amazonaws.com
        Validity
            Not Before: May 17 02:49:26 2018 GMT
            Not After : May 16 02:49:26 2021 GMT
        Subject: CN=internal-tsbb10lb01-758855236.us-east-1.elb.amazonaws.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:86:c2:47:e5:eb:f4:fa:e4:e5:15:d2:c7:e2:6d:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                D0:BA:AA:A6:05:BA:E8:4E:64:78:A9:71:B5:6D:9F:E7:41:DA:9C:29
    Signature Algorithm: sha256WithRSAEncryption
         2e:6b:60:17:37:35:d1:5c:c8:f1:5d:46:9a:f0:46:86:35:0f:
...
```